### PR TITLE
[MicrosoftApiModule] Auth code reconfigurations

### PR DESCRIPTION
**Pull request by mmhw**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related
https://github.com/demisto/content/pull/28652

## Description
This is a contribution that was canceled by the contributor, and we will include it anyway.

## Must have
- [x] Tests

### CODE REVIEW

**Comment by mmhw** [comment]
Hi @ShirleyDenkberg Can you please take a look?
It's the same RN for everyone.
Thanks!

File: Packs/AzureActiveDirectory/ReleaseNotes/1_3_16.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Azure Active Directory Identity Protection  (Deprecated)
+
+- Fixed an issue where changes made to the *Authorization code* parameter were not being reflected in the integration code, resulting in the continued use of the first parameter.
```
Comment (ShirleyDenkberg) [inline Packs/AzureActiveDirectory/ReleaseNotes/1_3_16.md:6]:
```suggestion
Fixed an issue where changes made to the *Authorization code* parameter were not being reflected in the integration code, resulting in the continued use of the first parameter.
```
Remove the bullet from each of the RNs.  There is no need for a bullet when there is only one item.

**Comment by ShirleyDenkberg** [comment]
@mmhw Just remove the bullet from each one of the RNs.  There is no need for a bullet when there is only one item.

**Comment by ShirleyDenkberg** [comment]
@michal-dagan @yucohen @bziser Doc review completed.

File: Packs/ApiModules/Scripts/MicrosoftApiModule/MicrosoftApiModule_test.py [hunk 2]
```diff
@@ -717,11 +721,49 @@ def test_generate_login_url():
...
 
     result = generate_login_url(client)
 
     expected_url = f'[login URL](https://login.microsoftonline.com/{TENANT}/oauth2/v2.0/authorize?' \
                    f'response_type=code&scope=offline_access%20https://graph.microsoft.com/.default' \
                    f'&client_id={CLIENT_ID}&redirect_uri=https://localhost/myapp)'
     assert expected_url in result.readable_output, "Login URL is incorrect"
+
+
+def test_get_access_token_auth_code_reconfigured(mocker, requests_mock):
```
Comment (bziser) [inline Packs/ApiModules/Scripts/MicrosoftApiModule/MicrosoftApiModule_test.py:734]: please add `given when then`

**Review by bziser — APPROVED** [review]

**Pull request merged**